scripts/build/gen-containerd-config.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
```

Comment:
Add filehead and `set -e`:https://github.com/kuasar-io/kuasar/blob/main/examples/run_example_container.sh#L2-L16